packages/client/src/components/app/Layout.svelte

```diff
@@ -169,15 +169,15 @@
 <!-- svelte-ignore a11y-no-static-element-interactions -->
 <!-- svelte-ignore a11y-click-events-have-key-events -->
 <div
-  class="component {screenId} layout layout--{typeClass}"
+  class="component layout layout--{typeClass}"
   use:styleable={$component.styles}
```

Comment:
Just actually tested this, and realised we need to also remove the `use:styleable` helper here (it adds back in the class name, and it's not needed since we don't allow screen styles). Right now you still get the selected border around the screen because of this.

Sorry about so many reviews on this one.